src/main/java/com/dreamhorizon/enchantments/DHEnchantments.java

```diff
@@ -17,7 +17,13 @@
  */
 
 package com.dreamhorizon.enchantments;
-import com.dreamhorizon.enchantments.enchantments.*;
```

Comment:
IntelliJ changed this automatically for me last time I did enchants. Did intelliJ also do this automatically for you? Shouldn't we just keep it on *?

Reply:
I learned java with the: NEVER USE WILDCARD Imports. There are reasons for this you can find online, I prefer to use alphabetically sorted imports with java.util always being at the bottom.

Reply:
Im fine with this, just asking. I'll see if I can get IntelliJ to fuck off with wildcards.